workers/api-server/src/http/wallet.rs

```diff
@@ -282,19 +282,14 @@ impl TypedHandler for RefreshWalletHandler {
         _query_params: QueryParams,
     ) -> Result<Self::Response, ApiServerError> {
         let wallet_id = parse_wallet_id_from_params(&params)?;
-        let wallet = find_wallet_for_update(wallet_id, &self.state).await?;
+        find_wallet_for_update(wallet_id, &self.state).await?;
```

Comment:
dead code?

Reply:
no, this now just effectively checks that the wallet is present, it's necessary